fuzz/.gitignore

```diff
@@ -0,0 +1,3 @@
+target
+corpus
```

Comment:
Where is the corpus, don't we need a seed in order to fuzz? Or do you think we should let users add their own.